docs/Versioning.md

```diff
@@ -90,9 +77,12 @@ ONNX specification. The union of the operator sets specified by a given model MU
 
 How nodes bind to operator declarations is strictly defined and are designed to increase model compatibilitey across ONNX implementations (appealing to the conservative clause of the robustnes principle). 
 
-How ONNX implementations bind an operator declaration to specific implementation is outside the scope
-of this specification.  Implementations of ONNX MAY elect 
-to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle. 
+How ONNX implementations bind an operator declaration to specific implementation is outside the scope of this specification.
+Implementations of ONNX MAY elect to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle.
+
+Implementations and models MAY provide experimental operators by agreeing to use operators in an experimental domain.
```

Comment:
there is no experimental domain. it's just a flag on the regular domain.

Reply:
Well, the could do that ;) Sorry, these are really about extensions not versioning, I've started a section over in IR and they belong there.